e2e/dummy/sign-and-send.go

```diff
@@ -0,0 +1,76 @@
+package dummy
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls/transactor"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/rs/zerolog/log"
+)
+
+var DefaultTransactionOptions = transactor.TransactOptions{
+	GasLimit: 2000000,
+	GasPrice: big.NewInt(0),
+	Value:    big.NewInt(0),
+}
+
+type signAndSendTransactor struct {
+	TxFabric       calls.TxFabric
+	gasPriceClient GasPricer
+	client         calls.ClientDispatcher
+}
+
+func NewSignAndSendTransactor(txFabric calls.TxFabric, gasPriceClient GasPricer, client calls.ClientDispatcher) transactor.Transactor {
+	return &signAndSendTransactor{
+		TxFabric:       txFabric,
+		gasPriceClient: gasPriceClient,
+		client:         client,
+	}
+}
+
+func (t *signAndSendTransactor) Transact(to *common.Address, data []byte, opts transactor.TransactOptions) (*common.Hash, error) {
```

Comment:
Don't think we need this file at all. If we add this to the regular sign and send transaction it would look exactly like this.